automation/jenkins_build.sh

```diff
@@ -470,6 +470,4 @@ if [ "$deploy" = "yes" ]; then
 
 fi
 
-# Cleanup
-# Keep this after writing all artifacts
-rm -rf $WORKSPACE/build
+# Do the cleanup in a post-build step
```

Comment:
We don't want to have to do a post-build step on all tasks that use `jenkins_build.sh`, and this change breaks other build jobs.
I suggest you use the `jenkins-deploy` folder if there are artifacts you need from the `jenkins_build.sh` step.

Reply:
:+1: